build.gradle

```diff
@@ -113,8 +113,12 @@ task deployNodes(type: net.corda.plugins.Cordform, dependsOn: ['jar']) {
             "ch.cordalo:corda-common-contracts:$cordalo_version",
             "ch.cordalo:corda-common-workflows:$cordalo_version"
         ]
-
-
+        extraConfig = [
+                jvmArgs : [ "-Xmx1g"]
```

Comment:
would add "-Xms1g"

Reply:
not supposed to be committed as not working